scripts/time-filter-proto.js

```diff
@@ -11,7 +11,7 @@ const destinations = (__ENV.DESTINATIONS || '5000,10000,25000,100000')
 const scenarios = destinations.reduce((accumulator, currentDestinations) => {
     accumulator[`sending_${currentDestinations}_destinations`] = {
         executor: 'constant-vus',
-        duration: '5m',
+        duration: '1m',
```

Comment:
Does this reduce the duration of the benchmark? Could this be a cli parameter instead? Sustained benchmarks are useful in some cases.

Reply:
sorry, I forgot to rollback